src/k2/cmd/demo/rpc_service.cpp

```diff
@@ -50,9 +51,15 @@ struct GET_Response {
     K2_PAYLOAD_FIELDS(key, value);
 };
 
+struct Echo_Message {
```

Comment:
I didn't realize you were going to check this file in. you should leave this rpc_service demo as it was, and instead add a new demo  for an echo server.

Reply:
Reverted back to the previous version